src/core/schema.js

```diff
@@ -0,0 +1,117 @@
+var utils = require('../utils/');
+
+var coordinates = utils.coordinates;
+var types = module.exports.types = {};
+
+registerType('boolean', booleanParse, booleanStringify);
```

Comment:
Maybe.

I should probably rename the public API to `registerPropertyType` too.